### DISCUSSION TITLE
Processor settings

### DISCUSSION THREAD

**Discussion by diverswan9** [nagadomi/nunif#128] [Q&A] [answered]
Firstly, thanks for working on this and sharing with us... from a long-time 3D enthusiast!

Under Processor settings...

I don't have a very good understanding of 'depth batch size' and 'worker threads'. I'm not quite sure of the best settings to use. One PC I have is Nvidia RTX 3060,one is RTX 4090, and another is GTX 1050. I use all 3 to process vids+photos. Currently I have them at 4(db) and 2(wt).Any guidance would be appreciated. 

Cheers!

**Comment by nagadomi** [comment, marked as answer]
https://github.com/nagadomi/nunif/issues/77#issuecomment-2005919214 's comment should help.

**Comment by diverswan9** [comment]
Thanks, I hadn't thought to look in 'issues'
